container_pipeline/workers/build.py

```diff
@@ -102,7 +102,7 @@ def handle_build_failure(self, job):
             'logs_dir': job['logs_dir'],
             'project_name': get_project_name(job),
             'job_name': job['jobid'],
-            'TEST_TAG': job['TEST_TAG']}
+            'TEST_TAG': job['test_tag']}
```

Comment:
the `TEST_TAG` key should be unique everywhere ? can we settle on upper case letters for `TEST_TAG` key name ?